src/main/java/uk/nhs/digital/nhsconnect/nhais/service/edifact_to_fhir/Config.java

```diff
@@ -0,0 +1,19 @@
+package uk.nhs.digital.nhsconnect.nhais.service.edifact_to_fhir;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import uk.nhs.digital.nhsconnect.nhais.model.edifact.ReferenceTransactionType;
+
+import java.util.Map;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+@Configuration
+public class Config {
```

Comment:
class not used?

Reply:
used by spring thanks to the `@Configuration` annotation